site/app/templates/grading/simple/Display.twig

```diff
@@ -108,19 +130,13 @@
                 <th style="width:20px">#</th>
                 <th style="width:70px" scope="col">Section</th>
                 <th style="width:90px; text-align: left" scope="col">
-                    <a href="{{ grading_url }}?view={{ view_all ? 'all' : '' }}&sort=id" aria-label="sort by ID">
-                        <span class="tooltiptext" title="sort by User ID">User ID </span><i class="fas fa-sort"></i>
-                    </a>
+                    <span title="sort by User ID">User ID </span>
                 </th>
                 <th style="width:110px; text-align: left" scope="col">
-                    <a href="{{ grading_url }}?view={{ view_all ? 'all' : '' }}&sort=first" aria-label="sort by First Name">
-                        <span class="tooltiptext" title="sort by First Name">First Name </span><i class="fas fa-sort"></i>
-                    </a>
+                    <span title="sort by First Name">First Name </span>
                 </th>
                 <th style="width:110px; text-align: left" scope="col">
-                    <a href="{{ grading_url }}?view={{ view_all ? 'all' : '' }}&sort=last" aria-label="sort by Last Name">
-                        <span class="tooltiptext" title="sort by Last Name">Last Name </span><i class="fas fa-sort"></i>
-                    </a>
+                    <span title="sort by Last Name">Last Name </span>
```

Comment:
These span titles no longer make sense and should be removed or the links within the headers as a quick version of the dropdown restored.